version.json

```diff
@@ -0,0 +1,3 @@
+{
+  "version": "[EXAMPLE]"
```

Comment:
Should this be [EXAMPLE]? If it's meant to be a placeholder value I think "dev" or "DEVELOPMENT" would be more appropriate